src/chrome/extension/scripts/chrome_core_connector.spec.ts

```diff
@@ -122,7 +122,7 @@ describe('core-connector', () => {
         cmd: 'on', type: uProxy.Update.COMMAND_REJECTED
       });
       expect(chrome.browserAction['setIcon']).toHaveBeenCalledWith(
-        {path: "icons/offline-19.png"});
+        {path: "icons/NotLoggedIn_32.gif"});
```

Comment:
Is it possible to refer to the constants from UserInterface here, and in other files where you are writing out the full filename+path?